modules/pam_limits/pam_limits.8.xml

```diff
@@ -58,9 +58,21 @@
       files in the above directory are not parsed.
     </para>
     <para condition="with_vendordir">
-      If there is no explicitly specified configuration file and
-      <filename>/etc/security/limits.conf</filename> does not exist,
-      <filename>%vendordir%/security/limits.conf</filename> is used.
+      By default limits are taken from the <filename>/etc/security/limits.conf</filename>
+      config file or, if that one is not present, the file
+      <filename>%vendordir%/security/limits.conf</filename>.
+      Then individual <filename>*.conf</filename> files from the
+      <filename>/etc/security/limits.d/</filename> and
+      <filename>%vendordir%/security/limits.d</filename> directory are read.
```

Comment:
Since you are talking about two directories here, I suppose `directories are read` would be more correct.